src/server/replica.cc

```diff
@@ -593,13 +598,14 @@ error_code Replica::ConsumeRedisStream() {
 error_code Replica::ConsumeDflyStream() {
   // Set new error handler that closes flow sockets.
   auto err_handler = [this](const auto& ge) {
+    // Trigger acl-checker
+    waker_.notifyAll();
```

Comment:
This is so confusing that we have the same name waker_ in Replica and in DflyShardReplica.
will you please rename them so we can see clearly where we wake which fibers
something like replication_waker and shard_replication_waker